### PR TITLE
Add Pushsafer Notification Service

### PR DESCRIPTION
**Pull request by appzer**
### Motivation for changes:
- add Pushsafer Notification Service

### Detailed changes:
-  add flexget/plugins/notifiers/pushsafer.py

### CODE REVIEW

File: flexget/plugins/notifiers/pushsafer.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		vibration: <integer> (default is 0)
+        timetolive: <integer> (default: (none))
+
+    """
+    schema = {'type': 'object',
+              'properties': {
+                  'private_key': one_or_more({'type': 'string'}),
+                  'url': {'type': 'string'},
+                  'url_title': {'type': 'string'},
+				  'device': {'type': 'string'},
```
Comment (liiight) [inline flexget/plugins/notifiers/pushsafer.py]: Weird indentation

File: flexget/plugins/notifiers/pushsafer.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                  'icon': {'type': 'integer', 'default': 1, 'maximum': 98, 'minimum': 1},
+				  'sound': {'type': 'integer', 'maximum': 28, 'minimum': 0},
```
Comment (liiight) [inline flexget/plugins/notifiers/pushsafer.py]: Weird indentation

File: flexget/plugins/notifiers/pushsafer.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+				  'vibration': {'type': 'integer', 'default': 0},
```
Comment (liiight) [inline flexget/plugins/notifiers/pushsafer.py]: Weird indentation

Comment (appzer) [inline flexget/plugins/notifiers/pushsafer.py]: fixed

Comment (crawln45) [inline flexget/plugins/notifiers/pushsafer.py]: still not 100% fixed

Comment (appzer) [inline flexget/plugins/notifiers/pushsafer.py]: where?

File: flexget/plugins/notifiers/pushsafer.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    """
+    Example::
+
+      pushsafer:
+        private_key: <string> your private key (can also be a alias key) - Required
+        title: <string> (default: task name)
+        body: <string> (default: '{{series_name}} {{series_id}}' )
+        url: <string> (default: '{{imdb_url}}')
+        url_title: <string> (default: (none))
+	device: <string> ypur device or device group id (default: (none))
```
Comment (cvium) [inline flexget/plugins/notifiers/pushsafer.py]: .

File: flexget/plugins/notifiers/pushsafer.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        icon: <integer> (default is 1)
+	sound: <integer> (default is (none))
```
Comment (cvium) [inline flexget/plugins/notifiers/pushsafer.py]: .

File: flexget/plugins/notifiers/pushsafer.py [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+	vibration: <integer> (default is 0)
```
Comment (cvium) [inline flexget/plugins/notifiers/pushsafer.py]: .

File: flexget/plugins/notifiers/pushsafer.py [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+        timetolive: <integer> (default: (none))
+
+    """
+    schema = {'type': 'object',
+              'properties': {
+                  'private_key': one_or_more({'type': 'string'}),
+                  'url': {'type': 'string'},
+                  'url_title': {'type': 'string'},
+		  'device': {'type': 'string'},
```
Comment (cvium) [inline flexget/plugins/notifiers/pushsafer.py]: .

File: flexget/plugins/notifiers/pushsafer.py [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+                  'icon': {'type': 'integer', 'default': 1, 'maximum': 98, 'minimum': 1},
+		  'sound': {'type': 'integer', 'maximum': 28, 'minimum': 0},
```
Comment (cvium) [inline flexget/plugins/notifiers/pushsafer.py]: .

File: flexget/plugins/notifiers/pushsafer.py [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+		  'vibration': {'type': 'integer', 'default': 0},
```
Comment (cvium) [inline flexget/plugins/notifiers/pushsafer.py]: .

**Comment by gazpachoking** [comment]
@appzer It looks like many of your indentation problems are from using a tab character instead of spaces. You can usually set your text editor to make sure it's using spaces instead of tab characters when you use the tab key.

**Comment by appzer** [comment]
and now

**Comment by liiight** [comment]
Looking good. Care to update the wiki?

**Comment by appzer** [comment]
pushsafer wiki entry = done

**Comment by liiight** [comment]
hmm, can you link to it? can't find it

**Comment by appzer** [comment]
> you just need to login to the wiki at https://flexget.com and edit it from there.

Done

**Pull request merged**